mlir-tensorrt/executor/include/mlir-executor-c/Runtime/Runtime.h

```diff
@@ -215,6 +215,11 @@ static inline bool mtrtRuntimeClientIsNull(MTRT_RuntimeClient client) {
   return !client.ptr;
 }
 
+/// Returns null client.
+static inline MTRT_RuntimeClient mtrtRuntimeClientGetNull() {
+  return MTRT_RuntimeClient{nullptr};
```

Comment:
Use C `NULL`, not `nullptr`, for same reasons as above. It looks like we have a few other uses of `nullptr` in the file unrelated to your change which we should correct as well. You can leave them as-is or update them, up to you.

Reply:
Sure. I will fix other sites as well.